Mathlib/Data/Fin/Tuple/Basic.lean

```diff
@@ -1071,6 +1072,110 @@ theorem contractNth_apply_of_ne (j : Fin (n + 1)) (op : α → α → α) (g : F
 
 end ContractNth
 
+section Take
```

Comment:
This is already in a big file, perhaps move to `Mathlib/Data/Fin/Tuple/Take.lean`.